vscode/qsharp.schema.json

```diff
@@ -26,28 +26,50 @@
       "title": "Lints",
       "type": "array",
       "items": {
+        "title": "lint",
         "type": "object",
-        "properties": {
-          "lint": {
-            "type": "string",
-            "enum": [
-              "divisionByZero",
-              "needlessParens",
-              "redundantSemicolons",
-              "deprecatedNewtype",
-              "deprecatedSet",
-              "discourageChainAssignment",
-              "needlessOperation",
-              "deprecatedFunctionConstructor",
-              "deprecatedWithOperator",
-              "deprecatedDoubleColonOperator"
-            ]
+        "oneOf": [
```

Comment:
Do you have public-facing documentation somewhere that tells users how to specify lint settings in their Q# project files? If so please remember to update that documentation as well. If not, please find an appropriate place to add it.